src/app.rs

```diff
@@ -184,8 +184,15 @@ pub fn build() -> App<'static, 'static> {
                 .multiple(true)
                 .number_of_values(1)
                 .require_delimiter(true)
-                .possible_values(&["permission", "user", "group", "size", "date", "name"])
-                .default_value("permission,user,group,size,date,name")
+                .possible_values(&[
+                    "permission",
+                    "user",
+                    "group",
+                    "size",
+                    "date",
+                    "name",
+                    "name-with-symlink",
```

Comment:
don't think this is supposed to be here

Reply:
"name-with-symlink" ? For now it sadly is because the padding rules and the formatting change between a `name` and a `name-with-symlink`.  I will try to re-merge them but it require a generic way to give parameters to blocks.

I should be able to remove in a next commit inside this PR I hope